pkg/syncer/broker/syncer_test.go

```diff
@@ -89,19 +94,25 @@ var _ = Describe("Broker Syncer", func() {
 		close(stopCh)
 	})
 
-	When("a local resource is created then deleted in the local datastore", func() {
-		It("should correctly sync to the broker datastore", func() {
+	When("a local resource is created in the local datastore", func() {
+		JustBeforeEach(func() {
 			test.CreateResource(localClient, resource)
-
 			test.AwaitResource(brokerClient, resource.GetName())
+		})
+
+		It("should correctly sync to the broker datastore", func() {
 			test.VerifyResource(brokerClient, resource, config.BrokerNamespace, config.LocalClusterID)
+		})
 
-			Expect(localClient.ResourceInterface.Delete(resource.GetName(), nil)).To(Succeed())
-			test.AwaitNoResource(brokerClient, resource.GetName())
+		Context("and then deleted", func() {
+			It("should be deletes from the broker datastore", func() {
```

Comment:
nit: Can be updated in a different PR 
```suggestion
			It("should be deleted from the broker datastore", func() {
```

Reply:
arghhh! ☹️